install/scrutiny-install.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck
+# Co-Author: MickLesk (Canbiz)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+# Source: https://github.com/AnalogJ/scrutiny
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  sudo \
+  curl \
+  smartmontools  \
+  make \
+  mc
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Scrutiny WebApp"
+mkdir -p /opt/scrutiny/config
+mkdir -p /opt/scrutiny/web
+mkdir -p /opt/scrutiny/bin
+RELEASE=$(curl -s https://api.github.com/repos/analogj/scrutiny/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+echo "${RELEASE}" >/opt/scrutiny_version.txt
```

Comment:
`echo "${RELEASE}" >/opt/${APPLICATION}_version.txt`

Reply:
I just fixed this https://github.com/tteck/Proxmox/issues/3269 you used `${APP}` instead of `${APPLICATION}`